dashboard/views/deliveries.ejs

```diff
@@ -28,6 +28,53 @@
                                 <div class="col-md-3"></div>
                                 <div class="col-md-6">
                                     <button class="btn btn-round pull-right" type="submit" data-l10n-id="showResults"><div class="ripple-container"></div></button>
+                                    <div class="btn pull-right btn-round" onclick="downloadAssignmentDeliveries()" data-l10n-id="ExportAssignemnt" id="users-exportassignment">Export Assignment</div>
```

Comment:
Looks like this text is not translated.

Reply:
yeah I will update this in `dashboard/public/I10n/locales.ini`